Bulldozer.CSV/Maps/Individual.cs

```diff
@@ -656,6 +696,23 @@ private int LoadIndividuals( CSVInstance csvData )
                         }
                     }
 
+                    // Add any additional emails as search keys
+                    if ( !string.IsNullOrWhiteSpace( rowAlternateEmails ) )
+                    {
+                        var emailSearcKeyDVId = DefinedValueCache.Get( new Guid( Rock.SystemGuid.DefinedValue.PERSON_SEARCH_KEYS_EMAIL ) ).Id; 
+                        foreach ( var email in rowAlternateEmails.Split( new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries ) )
```

Comment:
Do we need to protect against duplicates? Check if UI allows duplicates or not.

Reply:
UI does allow for duplicates as well.  I went ahead and added a .Distinct filter on the incoming alternate emails to at least give some protection against passing the same email in more than once in the alternateemails field from the import file.